roles/kafka_controller/tasks/get_meta_properties.yml

```diff
@@ -20,9 +20,25 @@
   register: zoo_cluster
   when: kraft_migration|bool
 
-- name: Format Data Directory
-  shell: "{{ binary_base_path }}/bin/kafka-storage  format -t={{ clusterid }} -c {{ kafka_controller.config_file }}  --ignore-formatted"
+- name: Create SCRAM Credentials
+  set_fact:
+    scram_credentials: >-
+      {% if 'SCRAM-SHA-512' in kafka_sasl_enabled_mechanisms %}
```

Comment:
isnt this templating introducing extra spaces before and after the actual content?

Reply:
That is why we have trimmed the extra spaces when using these credentials, any other approaches?